src/handlers/default.ts

```diff
@@ -124,15 +124,15 @@ export class InngestCommHandler {
 
     this.fns = functions.reduce<Record<string, InngestFunction<any>>>(
       (acc, fn) => {
-        if (acc[fn.id]) {
+        if (acc[fn.id(this.name)]) {
```

Comment:
Nit: As one of the comments above, `const id = fn.id(this.name)`, then use it to avoid accidentally only changing one.